tests/smoke_tests/dejong_and_mackay_2022/test_fig_5.py

```diff
@@ -58,21 +58,13 @@ def test_fig_5(backend_class, plot=False):
 
     # assert
     peaks_expected = {
-        0: (33, 0.018),
-        30: (92, 0.011),
-        60: (305, 0.012),
-        180: (717, 0.015),
-        540: (717, 0.015),
+        0: (34, 0.019),
+        1200: (2839, 0.03),
+        3600: (2839, 0.03),
     }
 
     for j, step in enumerate(steps):
-        print(step)
         peak = np.argmax(data_y[j])
         np.testing.assert_approx_equal(
-            actual=data_x[peak], desired=peaks_expected[step][0], significant=2
-        )
-        np.testing.assert_approx_equal(
-            actual=data_y[j][peak] * settings.rho,
-            desired=peaks_expected[step][1],
-            significant=2,
+            actual=data_x[peak], desired=peaks_expected[step][0], significant=1
```

Comment:
@edejong-caltech was this change intended? It is lowering the assert resolution to 1 significant digit, and removing the check on peak height - only peak location is being checked as a result

Reply:
Yes, it was intended. The location and height of the peak can fall in one of two bins depending on the exact random number that is sampled, so I do not think it is reasonable to ask it to be correct to 2 significant digits. Similarly with removing the peak height; sometimes it is 0.028, sometimes it is 0.031 depending on the exact realization.